modules/apps/29-fee/keeper/escrow.go

```diff
@@ -97,8 +97,9 @@ func (k Keeper) distributePacketFeeOnAcknowledgement(ctx sdk.Context, refundAddr
 	// distribute fee for reverse relaying
 	k.distributeFee(ctx, reverseRelayer, refundAddr, packetFee.Fee.AckFee)
 
-	// refund timeout fee for unused timeout
-	k.distributeFee(ctx, refundAddr, refundAddr, packetFee.Fee.TimeoutFee)
+	// refund unused fees
```

Comment:
should we update godoc for this function to indicate that we refund the unused amount from the escrowed fee?